# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xosview/libx11.patch
# Copyright (C) 2012 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xosview-1.10/Makefile.vanilla	2012-09-23 09:45:06.000000000 +0200
+++ xosview-1.10/Makefile	2012-09-23 09:46:09.000000000 +0200
@@ -17,7 +17,7 @@
 # Required build arguments
 
 CPPFLAGS += $(OPTFLAGS) -I. -MMD
-LDLIBS += -lX11
+LDLIBS += $(shell pkg-config x11 --libs)
 
 OBJS = Host.o \
 	Xrm.o \
